data/newtabicons-content.js

```diff
@@ -9,37 +9,38 @@
 'use strict';
 
 function mouseOverListener(e) {
-  let thumb = e.target.previousElementSibling;
+  if (e.target.previousElementSibling) {
+    var thumb = e.target.previousElementSibling;
+  } else {
+    var thumb = e.target;
+  }
```

Comment:
So, I think I'ld prefer this written as:
```js
let thumb = e.target.previousElementSibling || e.target;
```
(It's shorter, and having the two var statements looks weird to me, because they're really the same variable.)